.github/workflows/sonarcloud.yml

```
@@ -13,176 +15,69 @@ jobs:
+          ./gradlew sonarqube \
+            -Dsonar.login=${{ secrets.SONAR_TOKEN }} \
+            --info
         env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          # GITHUB_TOKEN is read implicitly by SonarScanner for PR decoration
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-#added a comment to test the sonarcloud.yml file
\ No newline at end of file
+          # SONAR_TOKEN is passed explicitly via -Dsonar.login
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)